bin/serverless.js

```diff
@@ -8,25 +8,29 @@ const logError = require('../lib/classes/Error').logError;
 const uuid = require('uuid');
 const initializeErrorReporter = require('../lib/utils/sentry').initializeErrorReporter;
 
-Error.stackTraceLimit = Infinity;
+(() => {
```

Comment:
No need for IIFE, in CJS it's totally valid to use `return` in module body

Reply:
oh it was already there. I just moved things a bit. I'd rather not change that in this PR. It has been here since forever.

Reply:
True, although it's totally useless (just confusing) in original form :) Safe to be removed. but never mind :)